### ISSUE TITLE
Unwrap `andThen` when adding `await`

### ISSUE THREAD

**Issue by scalvert** [simonihmig/ember-native-dom-helpers-codemod#25]
Currently, when the codemod runs and adds `await`, it should ideally remove the use of `andThen`, as these things seem semantically incompatible.

@rwjblue

**Comment by simonihmig** [comment]
@scalvert it should actually do just that! 
If you take a look at the test fixtures for acceptance tests, this is the case. E.g. [click.input.js](https://github.com/simonihmig/ember-native-dom-helpers-codemod/blob/master/__testfixtures__/acceptance/click.input.js) and [click.input.js](https://github.com/simonihmig/ember-native-dom-helpers-codemod/blob/master/__testfixtures__/acceptance/click.output.js).

Do you have an example where this is not working?

**Comment by rwjblue** [comment]
tldr; `andThen(() => {})` was not being transformed. I submitted https://github.com/simonihmig/ember-native-dom-helpers-codemod/pull/26 to fix.

**Comment by simonihmig** [comment]
Released as 0.2.1! Thanks again!
